### ISSUE TITLE
Config JSON: Swap status 1 and 2

### ISSUE THREAD

**Issue by ithinkandicode** [GodotModding/godot-mod-loader#131]
I think the valid status codes (0/1) should grouped, so that `statusCode > 1` always means an error. 

Atm to check if a status code is OK, you have to check for "_is status `0` or `2`_", which isn't very elegant.

**Current status codes:**

| #   | Description | Data |
| --- | ----------- | ---- |
| `0` | No errors   | *As requested* |
| `1` | Invalid `mod_id` | `{}` |
| `2` | No custom JSON exists | Defaults from *manifest.json* |
| `3` | Invalid `key`. Custom JSON does not exist | `{}` |
| `4` | Invalid `key`. Custom JSON *does* exist | `{}` |

**Proposed:**

| #   | Description | Data |
| --- | ----------- | ---- |
| `0` | No errors   | *As requested* |
| `1` | No custom JSON exists | Defaults from *manifest.json* |
| `2` | Invalid `mod_id` | `{}` |
| `3` | Invalid `key`. Custom JSON does not exist | `{}` |
| `4` | Invalid `key`. Custom JSON *does* exist | `{}` |

**Comment by Qubus0** [comment]
I feel like "no custom json" is not relevant enough to be part of the status codes. Just put an info log where it is set instead and return OK

**Comment by ithinkandicode** [comment]
@Qubus0 It's there because I experienced its usefulness first-hand while debugging, testing and using it. It tells you whether or not a custom JSON file exists. It's the same reason there's two status codes for an invalid key. Without these status codes, if you wanted to check if a custom JSON file exists, you'd have to implement your own way of checking for it. 

So they provide info that's potentially incredibly helpful, all via a single API call, with info that you'd otherwise have to check for manually.

As for the logging itself, I've raised a separate issue related to that:

- #133
